### PR TITLE
Add license 2

### PR DESCRIPTION
**Pull request by ahcodes**
## Fixes Issue

This PR fixes the following issues :
# Changes the license to MIT.

## Changes proposed.

No changes proposed.

## Check List (Check all the boxes which are applicable)

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain copyrighted content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots 
Add all the screenshots which support your changes

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,674 +1,21 @@
...
-
-  The GNU General Public License does not permit incorporating your program
-into proprietary programs.  If your program is a subroutine library, you
-may consider it more useful to permit linking proprietary applications with
-the library.  If this is what you want to do, use the GNU Lesser General
-Public License instead of this License.  But first, please read
-<https://www.gnu.org/licenses/why-not-lgpl.html>.
+MIT License
+
+Copyright (c) 2021 ahcodes
```
Comment (kaiwalyakoparkar) [inline LICENSE:3]: This is the license for your name. I would like of my name. You can refer the one of my MIT Licenses here: https://github.com/kaiwalyakoparkar/Find-Me-Resource/blob/main/LICENSE If you want then you can copy-paste it too.

**Review by kaiwalyakoparkar — CHANGES_REQUESTED** [review]
Name change in the license

File: LICENSE (hunk 1 above)
Comment (kaiwalyakoparkar) [reply LICENSE:3]: Or replace it with `Copyright (c) 2021 Kaiwalya Koparkar`

Comment (ahcodes) [reply LICENSE]: I updated the file as you requested. Let me know if anything else is amiss. ;)

**Review by kaiwalyakoparkar — APPROVED** [review]
LGTM :rocket: Thankyou very much for your time and contributions :smile:

**Comment by ahcodes** [comment]
> 
> 
> LGTM rocket Thankyou very much for your time and contributions smile

Sure, let me know if there's anything else I can help with. I'm a newbie, so doing stuff like this is a good learning experience!

**Comment by kaiwalyakoparkar** [comment]
@ahcodes Sure! You can check my other project which may fascinate you. If you want guidance related to OSS then you can join EddieHub (You can find all the information about it on my profile's organization section) or check out Eddie Jaoude's YT channel for more information about the community.

**Comment by ahcodes** [comment]
> 
> 
> @ahcodes Sure! You can check my other project which may fascinate you. If you want guidance related to OSS then you can join EddieHub (You can find all the information about it on my profile's organization section) or check out Eddie Jaoude's YT channel for more information about the community.

Hey sorry for the late response, had some wonky notifications settings. And yeah, I'll check the organization and his channel out, thanks!

**Pull request merged**